SECURITY.md

```diff
@@ -5,8 +5,9 @@
 | Version | Supported |
 | ------- | --------- |
 | Latest beta or release candidate    | ✅        |
+| v0.9.x  | ✅        |
 | v0.8.x  | ✅        |
-| < v0.8  | ❌        |
+| < v0.9  | ❌        |
```

Comment:
Seems incorrect, this says that we do not support anything less than v0.9. It should be `< v0.8`

Reply:
derp oh yeah sorry